dotnetv3/cross-service/README.md

```diff
@@ -13,4 +36,27 @@ The application uses the following services:
   - Amazon Simple Notification Service (Amazon SNS)
   - Amazon Translation Service
```

Comment:
If we're referring to the AWS translation service, it's "Amazon Translate".

Reply:
Fixed.